### PR TITLE
Move azureml folder to contrib folder

### PR DESCRIPTION
**Pull request by likebupt**
### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: contrib/azureml/azureml_designer_modules/module_specs/map.yaml [hunk 1]
```diff
@@ -58,12 +58,12 @@ outputs:
 code:
   ../../../../
```
Comment (miguelgfierro) [inline contrib/azureml/azureml_designer_modules/module_specs/map.yaml:59]: hey @likebupt are you sure that this path works?

**Comment by miguelgfierro** [comment]
@likebupt would you mind to add a short description in contrib/README?

**Comment by miguelgfierro** [comment]
Is the file `contrib/azureml/__init__.py ` necessary?

**Comment by likebupt** [comment]
@miguelgfierro  seems the script recommenders/sar_singlenode.py has been modified before, and delete the "normalize" argument.
I need to modify the module entry script and yaml spec.
Will test and then create another PR later this week.

**Pull request closed without merging**